config.base.yaml

```diff
@@ -29,13 +29,16 @@ alerts:
   emailalert:
     plugin: cloudmarker.alerts.emailalert.EmailAlert
     params:
+      is_ssl_required: True
```

Comment:
I believe this should be renamed to `use_ssl` considering the parameter name has changed in `EmailAlert` initializer.